cmd/minishift/cmd/config/validations.go

```diff
@@ -38,7 +38,9 @@ func IsValidDriver(string, driver string) error {
 }
 
 func RequiresRestartMsg(string, string) error {
-	fmt.Fprintln(os.Stdout, "To apply the changes, you must delete the current VM with `minishift delete` and start a new VM with `minishift start`.")
+	fmt.Fprintln(os.Stdout, "To apply the changes, "+
+		"you must delete the current VM with `minishift delete` if VM exist "+
```

Comment:
There are several ways to go about finding out whether the VM exits.

- Without modifying the current callback signature
  - You can just do what we do in other commands as well. Directly create the api `libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))`
  - We can put something into root.go. In _persistentPreRun_ we determine once for all commands whether the machine exists and set a variable accordingly. This variable can be queried by the other commands. AS a side effect we can remove some VM existence checks and we also get this into one single place. 
- with changing the callback signature
   - You can add additional parameters to the callback method, for example the name and of the VM. This way we stop the proliferation of using the constants file.

Either way, there are several ways to do that.